transform/transformers/ImageTransformer.py

```diff
@@ -1,43 +1,69 @@
-import os
-import glob
-import subprocess
-import zipfile
+#!/usr/bin/env python
+#   coding: UTF-8
+
+import argparse
 import datetime
 import dateutil.parser
-from transform import settings
-import shutil
+import glob
 from io import BytesIO
-from .PDFTransformer import PDFTransformer
-from transform.views.image_filters import get_env, format_date
+import itertools
+import json
+import logging
+import os.path
+import shutil
+import subprocess
+import sys
+import zipfile
+
 from requests.packages.urllib3.exceptions import MaxRetryError
+
+try:
```

Comment:
Depends how it's called:

`$ ~/py3.5/bin/python -m transform.transformers.ImageTransformer -h` is OK,

but...

`$ ~/py3.5/bin/python transform/transformers/ImageTransformer.py -h
Traceback (most recent call last):
  File "transform/transformers/ImageTransformer.py", line 25, in <module>
    from transform import settings
ModuleNotFoundError: No module named 'transform'`